src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -44,7 +44,7 @@ public class EditCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com "
-            + PREFIX_COURSE + "CS2103T,CS2101";
+            + PREFIX_COURSE + "CS2103T;CS2101";
```

Comment:
I think this (and other usages of explicit semicolons) could be changed in future to use `DEFAULT_DELIMITER` instead of an explicit semicolon, such as in the message usage string for `DeleteCommand`.

However, seeing as deadlines are tight, this is low priority.